scalafix-core/src/main/scala/scalafix/rewrite/NoXml.scala

```diff
@@ -0,0 +1,42 @@
+package scalafix
+package rewrite
+
+import scala.meta._
+import patch.TreePatch.AddGlobalImport
+
+case object NoXml extends Rewrite {
+
+  override def rewrite(ctx: RewriteCtx): Patch = {
+    val patch = ctx.tree.tokens.collect {
+      case tok @ Token.Xml.Start() =>
+        ctx.addLeft(tok, "xml\"\"\"")
+
+      case tok @ Token.Xml.End() =>
+        ctx.addRight(tok, "\"\"\"")
+
+      case tok @ Token.Xml.SpliceStart() =>
+        ctx.addLeft(tok, "$")
+
+      case tok @ Token.Xml.Part(part) =>
+        if (part.contains('$'))
+          ctx.replaceToken(tok, part.replaceAllLiterally("$", "$$"))
+        else Patch.empty
+
+    }.asPatch
+
+    // Emit warning on `{{`
```

Comment:
Either way, we should probably make it possible to test linter warnings like they do in scala/scala 

```scala
>>>
object a {
  xml"<foo>{{" // scalafix warn: no longer needed
}
```
that comment would then get remove in the diff and turn into check.